### DISCUSSION TITLE
Clipping predictions

### DISCUSSION THREAD

**Discussion by deven-gqc** [timeseriesAI/tsai#654] [Q&A]
Hello people, I’m using tsai to make predictions on a dataset and I can’t figure out the reason behind the output of the model.

If you take a look at the image below, the output of the predicted model keeps getting flattened out. What could be the reason behind this? Is it the loss function? or the architecture?

![image](https://user-images.githubusercontent.com/86305049/208483992-7c4f0606-ffc4-4b38-9d7e-ccaf7d420ce4.png)
Here is the code and the shapes of the inputs
![image](https://user-images.githubusercontent.com/86305049/208484071-c5df3af6-ba0b-43d7-8354-730ef585ad2f.png)
![image](https://user-images.githubusercontent.com/86305049/208484090-52b3c193-6f75-461a-87e7-13fada52be37.png)

```py
batch_tfms = [TSStandardize(by_sample=True)]
learn = TSRegressor(X, y, batch_tfms=batch_tfms, splits=split, 
                    arch=InceptionTimePlus, metrics=mae, bs=params['bs'], train_metrics=True)
lr = learn.lr_find()
learn.fine_tune(100, lr[0])
```

**Comment by oguiza** [comment]
Hi @deven-gqc,
Sorry for my late reply. 
There's no default clipping applied in tsai. 
I've tried reproducing your issue with dummy data but haven't been able to.
Here's a [gist](https://colab.research.google.com/gist/oguiza/b13cac471e2618063713560cbc476b50/scratchpad.ipynb) I've used.
